src/ToggleConfig.php

```diff
@@ -38,7 +46,12 @@ public function __construct(array $config)
 
         $this->apiEnabled = $config['api_enabled'];
         $this->apiPrefix = $config['api_prefix'];
+
         $this->driver = (string) $config['driver'];
+        /** @var array<string> $driverOptions */
+        $driverOptions = $config['driver_options'] ?? [];
```

Comment:
Maybe we should assert driver options when the driver value is "chain". I think with the current approach you can put an empty array in the chain driver options when at least one value is required.

Reply:
The assert is inside the `assertDriver` function. There, if the driver is `chain`, there're extra assertions to ensure the user pass the `driver_options` with valid values. This is only for those cases when we're using `inmemory` or `dbal` where there's no `driver_options` to define. In this case, the `driver_options` can be an empty array to avoid nullable field in the getter. What do you think?